Remove.py

```diff
@@ -1,35 +1,48 @@
 import PrintList
 
 
-def remove_1(to_Do, removed):
+def remove_1(to_Do):
 
   print("""
   ------------------------------
   Remove an item
   ------------------------------
   """)
-  
+
+  # Print current list
   PrintList.print_list(to_Do)
 
-  X = input("[Enter X to exit this screen]")
+  #User presses X to exit screen
+  X = input("[Enter X to exit this screen]").upper()
 
-  item_to_remove = int(input("What item would you like to remove?")) - 1
+  if X == "X":
+    return
+  else:
+    pass
 
-  
-  # Calling the item by index and not by name
-  if item_to_remove <= len(to_Do):
+  for item_to_remove in range(3):
```

Comment:
Why `range(3)` here?

Reply:
We agreed to give the user only 3 chances to remove or prioritize an item

Reply:
That's right, that's right. To remove confusion here, let's change `item_to_remove` in the for loop statement to something that better describes what that variable is. Maybe `user_attempt`?